### ISSUE TITLE
Is there anyway to sell permissions?

### ISSUE THREAD

**Issue by KritoKiller** [itsnewtjam/newtShop-Issues#39]
Is there anyway to sell permissions? Like if I wanted to sell /craft for the economy money, is there a way to do this?

**Comment by itsnewtjam** [comment]
Yes! You could use a command item to grant the permission. Basically just set a command item to run the command to grant the permission to the player. Info on command items [here](https://docs.newtjam.com/newtshop/item-format/#command-items).

**Comment by KritoKiller** [comment]
Worked flawlessly, thank you! I should have read the page with the info more thoroughly.

**Comment by KritoKiller** [comment]
> Yes! You could use a command item to grant the permission. Basically just set a command item to run the command to grant the permission to the player. Info on command items [here](https://docs.newtjam.com/newtshop/item-format/#command-items).

Quick questions. Is there a way to make it so after they buy the command item from the shop (let say the command newtshow.sellall) is there a way to stop them from beable to buy anymore since it is taking the money away but the perms are still the same.
